crowdfunding_compassion/controllers/project_controller.py

```diff
@@ -69,8 +69,33 @@ def _prepare_project_values(self, project, **kwargs):
         # Chronological list of sponsorships and fund donations for impact display
         impact = sorted(sponsorships + donations, key=lambda x: x["date"])
 
-        return {"project": project, "impact": impact}
+        fund = request.env['product.template'].sudo().search([
```

Comment:
```suggestion
        fund = project.product_id
```
You could even directly use it like that in the template and not really need to pass it as a separate variable...